Library/Formula/gmp.rb

```diff
@@ -45,7 +45,7 @@ def install
         return 0;
       }
     EOS
-    system ENV.cc, "test.c", "-L#{lib}", "-lgmp", "-o", "test"
+    system ENV.cc, "test.c", "-L#{lib}", "-I#{include}", "-lgmp", "-o", "test"
```

Comment:
Stylistically, I'd prefer
```ruby
system ENV.cc, "-I#{include}", "test.c", "-L#{lib}", "-lgmp", "-o", "test"
```
As this patch is not specific to Linux, please send it upstream to https://github.com/Homebrew/homebrew
See https://github.com/Linuxbrew/linuxbrew/blob/master/CONTRIBUTING.md